docs/notebooks/installation.rst

```diff
@@ -1,77 +1,167 @@
-Installation for Jupyter and JupyterLab
-=======================================
+Installing neptune-notebooks 
+============================
+This page provides instructions for installing neptune-notebooks -- the Neptune Jupyter extension. 
 
-Jupyter
--------
-Install `neptune-notebooks extension <https://github.com/neptune-ai/neptune-notebooks>`_:
+The extension integrates between Neptune and Jupyter or JupyterLab. When the extension is successfully installed, 
```

Comment:
`The extension integrates between Neptune and Jupyter or JupyterLab` - is this correct? I don't think I've ever heard such phrase.

Reply:
I am trying to say that the extension enables the integration or delivers the integration or produces/effects the integration. Is that not the correct idea?